articles/quickstart/native/react-native/00-login.md

```diff
@@ -16,8 +16,6 @@ budicon: 448
 
 <%= include('_includes/_intro') %>
 
-<%= include('_includes/_config') %>
-
```

Comment:
This was here because if you see the [final document](https://auth0-docs-content-pr-5087.herokuapp.com/docs/quickstart/native/react-native) there's a "Create client" step, which should be followed by getting the client_id/domain values and setting up the callback url, because you're already there in the client dashboard. I'd refactor the document to keep it that way. The rest looks good 👍